EPRN.Portal/Services/PRNService.cs

```diff
@@ -135,5 +135,12 @@ public async Task<PrnSavedAsDraftViewModel> GetDraftPrnConfirmationModel(int id)
                 PrnNumber = "PRN222019EFGF",
             };
         }
+
+        public async Task<ActionPrnViewModel> ActionPrnViewModel()
+        {
+            return new ActionPrnViewModel
```

Comment:
Model is not populated with anything. If that's correct, it might be useful to have a comment explaining why. But if nothings needed then chances you don't need to instantiate an object at all.

Reply:
Done.